components/dialog/stories/dialog.stories.js

```diff
@@ -65,17 +110,159 @@ export default {
 	},
 };
 
+const ExampleContent = Typography({
+	semantics: "body",
+	size: "m",
+	content: [
+		"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Auctor augue mauris augue neque gravida. Libero volutpat sed ornare arcu. Quisque egestas diam in arcu cursus euismod quis viverra. Posuere ac ut consequat semper viverra nam libero justo laoreet. Enim ut tellus elementum sagittis vitae et leo duis ut. Neque laoreet suspendisse interdum consectetur libero id faucibus nisl."
+	]
+});
+
+const ExampleButtons = [{
+	variant: "secondary",
+	treatment: "outline",
+	label: "Remind me later"
+}, {
+	treatment: "fill",
+	label: "Enable",
+	variant: "accent"
+}];
+
 export const Default = Template.bind({});
 Default.args = {
-	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+	heading: "Disclaimer",
```

Comment:
This very long heading was messing with the grid for dialogs, and really undoing the layout we were trying to create I think. I figured changing this heading was easier than editing the grid or something. And I also thought a dialog in the wild probably would have a heading that's 8 words long. 🤞 

<img width="504" alt="Screenshot 2024-06-11 at 1 31 40 PM" src="https://github.com/adobe/spectrum-css/assets/69602589/4eadd005-1587-4169-b739-268e37c516ea">

<img width="510" alt="Screenshot 2024-06-11 at 1 30 59 PM" src="https://github.com/adobe/spectrum-css/assets/69602589/50a4b30c-2bc0-4fd4-ae72-340067f410b8">

Reply:
Definitely leave the longer heading. That's there to test wrapping text and catch these sort of things. If this is causing an issue with the grid than we don't want to hide it. Oddly [I'm not seeing the text wrapping issue on the older version of what's built on prod currently](https://opensource.adobe.com/spectrum-css/preview/index.html?path=/story/components-dialog--default), so I think there may have been a regression recently? Something that went into main?

Reply:
Ok that makes sense- I didn't really think about the wrapping being used purposely like that. Reverted back to the longer heading and removed the buttons for now. I made another note to look into the buttons.

https://github.com/adobe/spectrum-css/pull/2833/commits/96464ed463b8c34341f7614d82e4eafdc07b2df6